fossa/analyze/action.yml

```diff
@@ -36,5 +36,11 @@ runs:
           --strict \
           --config "${CONFIGURATION_FILE}" \
           --branch "${BRANCH}" \
-          --revision "${REVISION_ID}"
+          --revision "${REVISION_ID}" 2>&1 | tee fossa_analyze_output.log
```

Comment:
💭 Not sure whether it could cause interference. If in doubt, we could place the file in `$RUNNER_TEMP` which points to a temp directory. Caveat: on self-hosted runners, this directory may not exist yet.